ChangeLog

```diff
@@ -34465,7 +34465,7 @@ Changes in version 0.2.0.19-alpha - 2008-02-09
   o Code simplifications and refactoring:
     - Remove some needless generality from cpuworker code, for improved
       type-safety.
-    - Stop overloading the circuit_t.onionskin field for both "onionskin
+    - Stop overloading the circuit_t.anonskin field for both "onionskin
```

Comment:
would keep it onionskin unless the variable itself has changed (e.g., is it anonrc?)